src/Core/src/Core/IEditor.cs

```diff
@@ -5,6 +5,11 @@
 	/// </summary>
 	public interface IEditor : IView, IText
 	{
+		/// <summary>
+		/// Gets a value that indicate if can modify the text.
+		/// </summary>
+		bool IsReadOnly { get; set; }
```

Comment:
This property must be in `ITextInput` (when we have all the properties implemented).